ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/identity_provider_config_service.proto

```diff
@@ -0,0 +1,159 @@
+  rpc CreateIdentityProviderConfig (CreateIdentityProviderConfigRequest) returns (CreateIdentityProviderConfigResponse);
+
+  // Get the identity provider configuration data by id.
+  // Errors:
+  // - ``NOT_FOUND``: if the identity provider configuration doesn't exist
+  // - ``UNAUTHENTICATED``: if the request does not include a valid access token
+  // - ``PERMISSION_DENIED``: if the claims in the token are insufficient to perform a given operation
+  // - ``INVALID_ARGUMENT``: if the payload is malformed or is missing required fields
+  rpc GetIdentityProviderConfig (GetIdentityProviderConfigRequest) returns (GetIdentityProviderConfigResponse);
+
+  // Update selected modifiable attribute of a identity provider config resource described
+  // by the ``IdentityProviderConfig`` message.
+  // Errors:
+  // - ``NOT_FOUND``: if the identity provider configuration doesn't exist
+  // - ``UNAUTHENTICATED``: if the request does not include a valid access token
+  // - ``PERMISSION_DENIED``: if the claims in the token are insufficient to perform a given operation
+  // - ``INVALID_ARGUMENT``: if the payload is malformed or is missing required fields
+  rpc UpdateIdentityProviderConfig (UpdateIdentityProviderConfigRequest) returns (UpdateIdentityProviderConfigResponse);
+
+  // List all existing identity provider configurations.
+  // Errors:
+  // - ``UNAUTHENTICATED``: if the request does not include a valid access token
+  // - ``PERMISSION_DENIED``: if the claims in the token are insufficient to perform a given operation
+  // - ``INVALID_ARGUMENT``: if the payload is malformed or is missing required fields
+  rpc ListIdentityProviderConfigs (ListIdentityProviderConfigsRequest) returns (ListIdentityProviderConfigsResponse);
+
+  // Delete an existing identity provider configuration.
+  // Errors:
+  // - ``NOT_FOUND``: if the identity provider configuration doesn't exist
+  // - ``UNAUTHENTICATED``: if the request does not include a valid access token
+  // - ``PERMISSION_DENIED``: if the claims in the token are insufficient to perform a given operation
+  // - ``INVALID_ARGUMENT``: if the payload is malformed or is missing required fields
+  rpc DeleteIdentityProviderConfig (DeleteIdentityProviderConfigRequest) returns (DeleteIdentityProviderConfigResponse);
+
+}
+
+message IdentityProviderConfig {
+  // The identity provider config identifier
+  // Must be a valid LedgerString (as describe in ``value.proto``).
+  // Required
+  string id = 1;
+
+  // When set, then the user identified by the identity provider is denied all access to the Ledger API.
+  // Otherwise, the user identified by the identity provider has access to the Ledger API as per the user's rights.
+  // Optional,
+  // Modifiable
+  bool is_deactivated = 2;
+
+  // Specifies the issuer of the JWT token that should be used in the "iss" claim of a JWT token to instruct the
+  // participant node to use this authentication domain to authenticate the token.
```

Comment:
I think we haven't defined what an "authentication domain" is

Reply:
I think I will just change to
```suggestion
  // participant node to use this identity provider to authenticate the token.
```